notes-app-backend/controllers/notes.js

```diff
@@ -26,11 +27,30 @@ notesRouter.post('/update', async (req, res) => {
     const n = await Note.findOne({ _id: id });
 
     n.note = note || n.note;
-    n.impotant = important !== undefined ? important : n.important;
+    n.important = important !== undefined ? important : n.important;
 
     const savedNote = await n.save()
 
     res.status(201).json(savedNote)
 })
 
+notesRouter.put('/important', async (req, res) => {
```

Comment:
This is a way of doing it but I think reusing the update function above works too.